nbs/tutorials/modular_nbdev.ipynb

```diff
@@ -1,5 +1,15 @@
 {
  "cells": [
+  {
+   "cell_type": "raw",
+   "id": "d3d59c9d-a2f2-4ee8-a05a-000d171aa01d",
+   "metadata": {},
+   "source": [
+    "---\n",
+    "skip_showdoc: true\n",
```

Comment:
Can you use the markdown frontmatter syntax here? I think we agreed at some point that its preferred over raw

Reply:
👍🏽